lib/python/treadmill_aws/sproc/garbage_collector.py

```diff
@@ -0,0 +1,67 @@
+"""Garbage collector plugin
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+import click
+from treadmill import plugin_manager
+from treadmill.cli import LIST
+from treadmill_aws.cli import handle_context_opt
+
+_LOGGER = logging.getLogger(__name__)
+_MODULE = 'treadmill_aws.garbage_collector'
+
+
+def _run_gc(gc_plugins):
+    """Garbage collector plugins executor.
+    """
+    for name in gc_plugins or plugin_manager.names(_MODULE):
+        try:
+            plugin = plugin_manager.load(_MODULE, name)
```

Comment:
I'm not a fan of implicitly loading the modules and executing them. Do we have any guarantee on what order they run in?

Reply:
we should assume that they are idempotent. cellsync works this way, i think it is ok.